Configuration/Yaml/FormSetup.yaml

```diff
@@ -17,6 +17,7 @@ imports:
   - { resource: "./FormElements/DatePicker.yaml" }
   - { resource: "./FormElements/FileUpload.yaml" }
   - { resource: "./FormElements/ImageUpload.yaml" }
+  - { resource: "./FormElements/Countryselect.yaml" }
```

Comment:
@mkarulin This type of field (countryselect) is coming from another package, right? formhelper, if I'm not mistaken. The thing is - form_fieldnames does not know about formhelper. Therefore it seems odd to add a name attribute to field type that is not officially known.

Having said that, I don't really have a better idea. We could create a new package formhelper_form_fieldnames which would add compatibility between formhelper and form_fieldnames. But this seems overkill to me. And we might also have to do it for other packages like formrelay and formhelper. We might want to find a good strategy in general.